test/intl402/DurationFormat/prototype/format/rounding-mode-trunc-for-seconds.js

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2024 Sosuke Suzuki. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.DurationFormat.prototype.format
+description: |
+  nfOptions.roundingMode for seconds / microseconds / nanoseconds should be "trunc"
+info: |
+  1.1.14 PartitionDurationFormatPattern
+    (...)
+    4. While numericUnitFound is false, repeat for each row in Table 2 in table order, except the header row:
+      e. If style is "numeric" or "2-digit", then
+        i. Append FormatNumericUnits(durationFormat, duration, unit, signDisplayed) to result.
+        (...)
+      f.Else,
+        (...)
+        ii. If unit is "seconds", "milliseconds", or "microseconds", then
+          (...)
+          f. Perform ! CreateDataPropertyOrThrow(nfOpts, "roundingMode", "trunc").
+
+    1.1.12 FormatNumericUnits
+      (...)
+      18. If secondsFormatted is true, then
+        a. Append FormatNumericSeconds(durationFormat, secondsValue, minutesFormatted, signDisplayed) to numericPartsList.
+        (...)
+
+    1.1.11 FormatNumericSeconds
+      (...)
+      15. Perform ! CreateDataPropertyOrThrow(nfOpts, "roundingMode", "trunc").
+      (...)
+locale: [en]
+includes: [testIntl.js]
+features: [Intl.DurationFormat]
+---*/
+
+const durations = [
+  // 1
+  {
+    fractionalDigits: 0,
+    duration: {
+      seconds: 1,
+      milliseconds: 500,
+    },
+  },
+  // 0.001
+  {
+    fractionalDigits: 3,
+    duration: {
+      milliseconds: 1,
+      microseconds: 500,
+    }
+  },
+  // 0.000001
+  {
+    fractionalDigits: 6,
+    duration: {
+      microseconds: 1,
+      nanoseconds: 500
+    }
+  }
+];
+
+for (const { fractionalDigits, duration } of durations) {
+  const df = new Intl.DurationFormat("en", { seconds: "numeric", fractionalDigits });
+  const expected = formatDurationFormatPattern(df, duration);
```

Comment:
I'd like to try to avoid adding new tests that use the "write a mini-polyfill inside test262" approach, if possible.

How about something like this? 

```js
for (const { numericValue, fractionalDigits, duration } of durations) {
  const df = new Intl.DurationFormat("en", { seconds: "numeric", fractionalDigits });
  const nf = new Intl.NumberFormat("en", { maximumFractionDigits: fractionalDigits, roundingMode: "trunc" });
  const expected = nf.format(numericValue);
  assert.sameValue(df.format(duration), expected);
}
```

Reply:
Thank you for reviewing. I've addressed your comments https://github.com/tc39/test262/pull/4115/commits/371f30d50d3bf0ed8a394d4500d11cb6bf63ad8f

Reply:
Oh sorry in the commit, I use invalid harness, I'll fix it later

Reply:
Fixed https://github.com/tc39/test262/pull/4115/commits/ac9629bceed054d12d4e8d210055ca1c40a7b978